ep_testing/tests/energyplus.py

```diff
@@ -27,8 +26,18 @@ def run(self, install_root: str, verbose: bool, kwargs: dict):
             os.symlink(eplus_binary, eplus_binary_to_use)
         else:
             eplus_binary_to_use = eplus_binary
-        try:
-            check_call([eplus_binary_to_use, '-D', idf_path], stdout=dev_null, stderr=STDOUT)
+
+        cmd = [eplus_binary_to_use, '-D', idf_path]
+        r = subprocess.run(cmd,
+                           stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        if r.returncode == 0:
             print(' [DONE]!')
-        except CalledProcessError:
-            raise EPTestingException('EnergyPlus failed!')
+        else:
+            raise EPTestingException(
+                'EnergyPlus failed!\n'
+                f'Command {cmd} failed with exit status {r.returncode}!\n'
+                'stderr:\n'
+                f'{r.stderr.decode().strip()}'
+                '\n\n'
+                'stdout:\n'
+                f'{r.stdout.decode().strip()}')
```

Comment:
Improve error reporting. I may need to also wrap the subprocess.run in a try block to catch "File not Found" and co